.github/dependabot.yml

```
@@ -0,0 +1,51 @@
+    versioning-strategy: increase
+    allow:
+      - dependency-name: "@nestia/fetcher"
+      - dependency-name: "typia"
+      - dependency-name: "tgrid"
+    groups:
+      Samchon:
+        patterns:
+          - "@nestia/fetcher"
+          - "typia"
+          - "tgrid"
+#######################################################
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)